src/routes/blog/post/encrypted-attributes-for-sensitive-fields/+page.markdoc

```diff
@@ -0,0 +1,96 @@
+
+In Appwrite, these fields can now be stored with encryption at rest by using **encrypted string attributes**. This feature allows developers to protect data inside their database without requiring complex encryption logic on the client side. It is not end-to-end encryption, and it does not prevent project owners from reading data. However, it introduces a meaningful boundary between sensitive values and the infrastructure that holds them.
+
+This guide introduces encrypted string attributes, explains how they function, where they are appropriate, and what limitations they carry.
+
+# A protection layer for sensitive data
+
+When a string attribute is marked as encrypted, Appwrite applies AES-128 encryption in Galois/Counter Mode (GCM) before writing it to the database. This means the data is never stored in plaintext at rest. It remains protected even in the case of unauthorized access to the underlying database. For many applications, that separation between what is readable in production and what is stored in the backend is enough to reduce risk and support stronger privacy standards.
+
+# Using encrypted attributes in the Console
+
+Encrypted string attributes can be created or edited through the Appwrite Console.
+
+To create an encrypted attribute:
+
+1. Open your Appwrite Console
+2. Navigate to **Databases** and select the relevant database
+3. Choose a collection and open the **Attributes** tab
+4. Click **Create attribute**, then select **String**
+
+    ![create attribute modal](/images/blog/encrypted-attributes-for-sensitive-fields/create-attribute-modal.png)
+
+5. In the attribute creation modal, enable the **Encrypted** checkbox
+
+    ![string attribute modal](/images/blog/encrypted-attributes-for-sensitive-fields/string-attribute-modal.png)
+
+
+You may also configure standard settings, such as size limits, default values, whether the field is required, or whether it supports arrays. Note that encrypted string attributes require a **minimum size of 150 characters**.
+
+Once created, the attribute will appear in your schema with a lock icon to indicate that its contents are encrypted at rest.
+
+![encrypted attribute screen](/images/blog/encrypted-attributes-for-sensitive-fields/encrypted-attribute-screen.png)
+
+Note that the encrypted string attribute feature is available only on **Pro plans and higher**.
+
+# What encryption changes, and what it does not
+
+Encryption changes how Appwrite stores a value, but it does not change how that value is accessed within the Console. Project owners and team members with sufficient access will still be able to read encrypted attribute values in plaintext. Encryption applies only to how the data is written to disk.
+
+Most importantly, **encrypted attributes cannot be queried**. You cannot filter, search, or sort based on encrypted values.
+
+If you encrypt a field that you later need to query, you will need to delete and recreate it.
```

Comment:
should we add this as an alert or in bold because `you will need to delete and recreate it`?